asllib/Parser.mly

```diff
@@ -587,6 +587,7 @@ let stmt :=
   )
 
 let stmt_list := ~ = list1(stmt) ; <stmt_from_list>
+let stmt_list0 == stmt_list | annotated({ S_Pass })
```

Comment:
nit: perhaps move this to just before line 738 (`let opn ...`), and mark `[@internal true]`?

Reply:
It's an inlined symbol, so it can't carry an attribute, and should be inlined before menhir2bnfc.

Reply:
Ah my mistake!